tests/difference/core/quint_model/ccv_statemachine.qnt

```diff
@@ -0,0 +1,506 @@
+// -*- mode: Bluespec; -*-
+module ccv_statemachine {
+    // A basic state machine that utilizes the CCV protocol.
+    import ccv_types.* from "./ccv"
+    import Time.* from "./libraries/Time"
+    import extraSpells.* from "./libraries/extraSpells"
+
+    pure val consumerChains = Set("consumer1", "consumer2", "consumer3")
+    pure val chains = consumerChains.union(Set(PROVIDER_CHAIN))
+    pure val unbondingPeriods = chains.mapBy(chain => 2 * Week)
+    pure val ccvTimeouts = chains.mapBy(chain => 3 * Week)
+
+    pure val nodes = Set("node1", "node2", "node3", "node4", "node5", "node6", "node7", "node8", "node9", "node10")
+    pure val InitialValidatorSet = nodes.mapBy(node => 100)
+
+    import ccv(VscTimeout = 5 * Week, CcvTimeout = ccvTimeouts, UnbondingPeriodPerChain = unbondingPeriods, ConsumerChains = consumerChains).* from "./ccv"
+
+
+    var currentState: ProtocolState
+
+
+    pure def ACTIONS = Set(
```

Comment:
I couldn't find where this is being used. If not, we can remove or have as comment instead on what the possible actions are?